dask/array/tests/test_array_core.py

```diff
@@ -414,6 +431,7 @@ def test_norm():
     assert b.vnorm(ord=4, axis=0, keepdims=True).ndim == b.ndim
```

Comment:
The problem here is that the axis keyword argument was not added to `numpy.linalg.norm` until numpy 1.8. so the first two `assert`s work, but the others do not.

Are you suggesting I break this into two tests, and only skipif one?